README.md

```diff
@@ -28,6 +28,26 @@ The search lib is modular and currently implements the following layers:
 
 ![Search Lib Architecture](./docs/images/layer_cake.png)
 
+## Versioning & release policy
+
+Each Vind released is labeled in the repository with a tag fitting the following schema: _vind-va.b.c_ where 
+**vind-v** just points out that this belongs to a Vind version and **a.b.c** represents the release number, also used 
+for the artifact version. Prior to **vind-v1.3.0** there was not a clear policy on versioning therefor, there was also not 
+an explanation about it. 
+From __1.3.0__ on the releases will strictly stick to the following diagram:
+
+![Versioning policy](./docs/images/versioning_policy.png)
```

Comment:
add on the first number add something like "this includes version changes where a complete reindex is necessary to keep the system working"